src/BlockTemplatesController.php

```diff
@@ -295,7 +295,7 @@ public function get_block_file_template( $template, $id, $template_type ) {
 	 * @return array
 	 */
 	public function add_block_templates( $query_result, $query, $template_type ) {
-		if ( ! BlockTemplateUtils::supports_block_templates() ) {
+		if ( ! BlockTemplateUtils::supports_block_templates( 'wp_template_part' ) ) {
```

Comment:
I think we'd benefit from a comment here on why `wp_template_part` for context.

Reply:
Yeah, I think you're right. I just checked the filter params to be sure it was always going to be one of the two (`wp_template_part` / `wp_template` ) and [it looks to be the case](https://github.com/WordPress/WordPress/blob/master/wp-includes/block-template-utils.php#L1044C56-L1044C56).

Reply:
I also re-tested and everything works as expected from what I can see unless the E2E tests pick up anything.